manim/utils/space_ops.py

```diff
@@ -91,7 +102,27 @@ def quaternion_mult(*quats):
         )
 
 
-def quaternion_from_angle_axis(angle, axis, axis_normalized=False):
+def quaternion_from_angle_axis(
+    angle: float, axis: np.ndarray, axis_normalized: bool = False
+) -> List[float]:
+    """Gets a quaternion from an angle and an axis.
+    For more information, check:
+    https://en.wikipedia.org/wiki/Conversion_between_quaternions_and_Euler_angles
```

Comment:
Can this be formatted into a proper link? The link text can be "this Wikipedia article" or something similar.